app/routes/oauth-stripe.js

```diff
@@ -0,0 +1,14 @@
+import Ember from 'ember';
+
+const {
+  Route
+} = Ember;
+
+export default Route.extend({
+  // model(params) {
```

Comment:
I'm actually thinking the following might be more explicit and easier to understand:

```javascript
model({ code, state }) {
  return this.get('store').createRecord('stripe-account', { code, state });
}
```

Thoughts?